react/week2/my-todos/src/index.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+const todos = [
+  {
+    "id": 1,
+    "description": "Get out of bed",
+    "checked": false
+  },
+  {
+    "id": 2,
+    "description": "Brush teeth",
+    "checked": false
+  },
+  {
+    "id": 3,
+    "description": "Eat breakfast",
+    "checked": false
+  },
+  {
+   "id": 4,
+   "description": "Learn React",
+   "checked": false
+  },
+  {
+    "id": 5,
+    "description": "Do homework",
+    "checked": false
+  }
+ ];
+ 
+class TodoList extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { 
+      todoList: []
+    } 
```

Comment:
In the version of react we're using, you can omit the constructor and super, it's done behind the scenes, so you could re-write this like: 
```suggestion
    state = { seconds: 0 };
```
You can read more about it in the [docs](https://reactjs.org/docs/react-component.html#constructor), however I will admit, they are a bit confusing to understand.